test/test_transforms_v2_refactored.py

```diff
@@ -2270,3 +2270,24 @@ def test_image_correctness(self, permutation, batch_dims):
         expected = self.reference_image_correctness(image, permutation=permutation)
 
         torch.testing.assert_close(actual, expected)
+
+
+class TestToPureTensor:
+    def test_correctness(self):
+        input = {
+            "img": make_image(),
+            "img_tensor": make_image_tensor(),
+            "img_pil": make_image_pil(),
+            "mask": make_detection_mask(),
+            "video": make_video(),
+            "bbox": make_bounding_box(),
+            "str": "str",
+        }
```

Comment:
Can we parametrize over that rather than looping below?

Reply:
A minor perk of doing it this way is that we also test the "arbitrary structure" support.